### PR TITLE
Updated landing page & video links

### PR DESCRIPTION
**Pull request by renebrandel**
*Issue #, if available:*
- New landing page with 90s explanation video
- Updated youtube links
- minor styling fixes

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: client/src/styles/color-effect-utils.scss [hunk 1]
```diff
@@ -113,3 +113,7 @@
 .text-align-center {
   text-align: center;
 }
+
+.align-self-center {
```
Comment (sammartinez) [inline client/src/styles/color-effect-utils.scss:117]: cool name :)

File: docs/cli/start/install.md [hunk 2]
```diff
@@ -29,10 +29,7 @@ To set up the Amplify CLI on your local machine, you have to configure it to con
 
 Watch the video below to learn how to install and configure the Amplify CLI or skip to the next section to follow the step-by-step instructions.
 
-<iframe
-  allowfullscreen
-  src="https://www.youtube.com/embed/fWbM5DLh25U"
-></iframe>
+<iframe src="https://www.youtube-nocookie.com/embed/fWbM5DLh25U" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (sammartinez) [inline docs/cli/start/install.md:32]: sweet!

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Review by mauerbac — APPROVED** [review]
thank you @renebrandel

**Comment by sammartinez** [comment]
Validation locally is complete. Thanks for tackling this @renebrandel

**Pull request merged**